src/conways-game-of-life/index.ts

```diff
@@ -0,0 +1,81 @@
+const updateCellStatus = (x: number, y: number, currentCellStatus: number, neighbourSum: number) => {
+    if (currentCellStatus === 1 && neighbourSum > 3) {
+        return 0;
+    }
+    else if (currentCellStatus === 1 && neighbourSum < 2) {
+        return 0;
+    }
+    else if (currentCellStatus === 0 && neighbourSum === 3) {
+        return 1;
+    }
+    else {
+        return currentCellStatus;
+    }
+} //Apply the conditions for updating the status of a cell 
+
+const calculateNeighbourSum = (grid: number[][], x: number, y: number) => {
+    let toBeSummed: number[] = [
```

Comment:
Why `let` and not `const`?

Reply:
My first idea was to create an array which would receive neighbours values one by one and that would have undefined elements removed from it later. So I thought let was more appropriate. Although I remember now that elements in a array can be added and removed even if the array is declared as a const.  I will change it in the code. Thanks.